src/Components/WebAssembly/testassets/HostedInAspNet.Client/wwwroot/seleniumworkaround.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+  const logs = [];
+
+  console.defaultLog = console.log.bind(console);
+  console.log = function () {
+    console.defaultLog.apply(console, arguments);
+    logs.push(Array.from(arguments).join('\n'));
```

Comment:
Why join with `\n`? If you run `console.log('a', 'b', 'c')`, the output is:

```
a b c
```

So wouldn't it be more normal to join with `' '`?